packages/components/react-web/src/ThemeProvider/ThemeProvider.js

```diff
@@ -0,0 +1,26 @@
+import React, { ThemeProvider as EmotionThemeProvider } from '@emotion/react';
+import PropTypes from 'prop-types';
+import { useContext, createContext } from 'react';
+import defaultTheme from "@compassion-gds/tokens";
+
+const ThemeContext = createContext(defaultTheme);
+
+export const useTheme = () => useContext(ThemeContext);
+
+export const ThemeProvider = ({ theme = defaultTheme, children }) => (
+    <ThemeContext.Provider value={theme}>
+        <EmotionThemeProvider theme={theme}>
+            {children}
+        </EmotionThemeProvider>
+    </ThemeContext.Provider>
+);
```

Comment:
We're providing the Emotion context for compatibility

The css prop can be a function that would receive the `theme` automatically from emotion

```jsx
<MyComponent css={theme => css`background: ${theme.background}`} />
```

Reply:
Love the broader abstraction brought to `ThemeProvider`.